contracts/v2/aggchains/AggchainECDSA.sol

```diff
@@ -236,4 +236,12 @@ contract AggchainECDSA is AggchainBase {
         // Emit event
         emit OnVerifyPessimisticECDSA(newStateRoot);
     }
+
+    /**
+     * @notice Function to retrieve the current version of the contract.
+     * @return version of the contract.
+     */
+    function version() external view override returns (string memory) {
```

Comment:
All this version function that are reading the version from a constant (bytecode) can be moved to `pure` and no need to add the `override` if they are not inheriting from a virtual funcion